docs/core/deploying/single-file.md

```diff
@@ -90,6 +90,39 @@ For example, add the following property to the project file of an assembly to em
 </PropertyGroup>
 ```
 
+## Publish a single file app - sample project file
+
+Here's a sample project file that specifies single-file publishing:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <PublishSingleFile>true</PublishSingleFile>
+    <SelfContained>true</SelfContained>
+    <RuntimeIdentifier>linux-x64</RuntimeIdentifier>
+    <PublishTrimmed>true</PublishTrimmed>
+    <PublishReadyToRun>true</PublishReadyToRun>
```

Comment:
Missed this, I don't think R2R is supported in linux-x64

Reply:
I changed the RID to win-x64